greeter-osm.conf

```diff
@@ -12,15 +12,15 @@ statusfile = ./statusgreeter
 subject = Přivítání
 mainmessage = Ahoj <nick>,
 
-	děkujeme za tvůj první příspěvek do OpenStreetMap. Zároveň si dovolujeme srdečně tě přivítat mezi komunitu nadšenců, kteří podobnými úpravami doplňují či zlepšují mapu už několik let. Pokud je toto spíše jednorázová záležitost, nevadí, možná ve svém okolí znáš někoho, kdo rád chodí do lesa, jezdí na kole nebo je zkrátka nadšenec do map - pak budeme rádi, když mu o OpenStreetMap řekneš. Způsobů, jakými je možné přispět, je mnoho - kromě samotného editování to může být například nahrávání GPS záznamů, nahlašování chyb nebo třeba propagace projektu.
+	děkujeme za tvůj první příspěvek do OpenStreetMap. Zároveň si tě dovolujeme srdečně přivítat mezi komunitu nadšenců, kteří podobnými úpravami doplňují či zlepšují mapu už několik let. Pokud je toto spíše jednorázová záležitost, nevadí, možná ve svém okolí znáš někoho, kdo rád chodí do lesa, jezdí na kole nebo je zkrátka nadšenec do map – pak budeme rádi, když mu o OpenStreetMap řekneš. Kromě samotného kreslení mapy lze pomáhat například nahráváním GPS záznamů, nahlašováním chyb, nebo třeba propagací OpenStreetMap.
 
-	Pokud se chceš na něco zeptat nebo se jenom podělit o své zkušenosti či nápad na zlepšení, tak tě rádi potkáme v české konferenci talk-cz@openstreetmap.org (https://lists.openstreetmap.org/listinfo/talk-cz?language=cs). Další informace o české komunitě: https://openstreetmap.cz/komunita 
+	Pokud se chceš na něco zeptat nebo se jenom podělit o své zkušenosti či nápad na zlepšení, rádi tě potkáme v české konferenci talk-cz@openstreetmap.org (https://lists.openstreetmap.org/listinfo/talk-cz?language=cs). Další informace o české komunitě: https://openstreetmap.cz/komunita 
 	A nezapomeň na zlaté pravidlo OpenStreetMap: bez výslovného povolení z cizích map (Mapy Google, Mapy.cz) nic nekopírujeme!
```

Comment:
Tohle bych vyhodil. Zbytečně to text natahuje a snižuje tak pravděpodobnost, že si to vůbec někdo přečte.
Pro nováčka mi přijde důležitější info o tom, že má mapovat to co existuje. Nečekám, že by drtivá většina nováčků přelévala data z jedné databáze do druhé, nebo vyjednávala povolení pro použití dalších datových zdrojů…

Reply:
Je to jedna věta. To zase tak nenatahuje :) U nováčků - a zejména u mladých nováčků - si nedělám iluze, že ví něco o licencích a autorském zákoně. A kopírování z jiných zdrojů bez svolení je bohužel docela čas^H^H^H^H^H ehm, není neobvyklé. :(

Reply:
OK, pokud nováčci velmi často kopírují _něco_ z jiných map/zdrojů tak je asi důležité ponechat. Pokud to tak není a je to tam jen do počtu, vyhodil bych to.
Z UX/CX testů nám vždy vycházelo kratší = lepší. Nechal jsem to tam, protože jsem si nebyl jistý, proč to tam vůbec je…